examples/experimental/wasm/server/api/sum.ts

```diff
@@ -1,7 +1,9 @@
-import { useQuery, lazyHandle } from 'h3'
+import { useQuery, defineLazyHandler } from 'h3'
 
-export default lazyHandle(async () => {
-  const { exports: { sum } } = await loadWasmInstance(
+export default defineLazyHandler(async () => {
+  const {
+    exports: { sum }
+  } = await loadWasmInstance(
```

Comment:
Let's keep formatting.

```suggestion
  const { exports: { sum } } = await loadWasmInstance(
```